restcomm/restcomm.ui/src/main/webapp/modules/numbers-incoming-details.html

```diff
@@ -88,7 +88,13 @@ <h4 class="number-advanced-label">USSD Fallback URL</h4>
         <a href="" class="number-advanced-toggle" ng-click="showUssdAdvanced = !showUssdAdvanced"><span ng-show="showUssdAdvanced">hide </span>advanced</a>
     </div>
 </div>
-
+<h4 class="voice-sms-ussd">Refer</h4>
+<div class="row number-infos-border">
+    <div class="col-md-2" style="padding-right: 0;">
+        <h4 class="voice-video-refer-request">Refer Request</h4>
+    </div>
+    <rc-endpoint-url details-loaded="numberDetails.$resolved" sid-var="numberDetails.refer_application_sid" url-only-var="false" method-var="numberDetails.refer_method" url-var="numberDetails.refer_url" apps="localReferApps"></rc-endpoint-url>
```

Comment:
If applications listed in Refer request are by definition voice application, why not use directly the `localVoiceApps` variable?

Thus the <rc-endpoint-url /...> will contains:
`apps="localVoiceApps"`